PlayFabClientSDK/pom.xml

```diff
@@ -1,12 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
FYI, this file is built from SdkGenerator, so I will decline this request and generate these changes there.